app/views/search/_results.html.erb

```diff
@@ -1,12 +1,12 @@
 <div class="petition_index_view">
   <%= render 'search/result_tabs' %>
   <div class="petition_list">
-    <% if @petition_search.petitions.present? -%>
+    <% unless @petitions.to_a.empty? -%>
```

Comment:
EXTREME NITPICK: We've delegated ``empty?`` in ``PetitionSearch`` to the ``petition_results``, so we can probably drop the ``to_a`` here.